api/report_server.thrift

```diff
@@ -42,6 +42,13 @@ struct RunReportCount{
 }
 typedef list<RunReportCount> RunReportCounts
 
+struct CheckerDataCount {
+  1: string             name,     // Checker name
```

Comment:
The nomenclature pull-request will make this file also be formatted as full sentences, this should be so from now on.